addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1207,7 +1207,7 @@ var SnippetsMenu = Widget.extend({
             this.invisibleDOMMap = new Map();
             const $invisibleDOMPanelEl = $(this.invisibleDOMPanelEl);
             $invisibleDOMPanelEl.find('.o_we_invisible_entry').remove();
-            const $invisibleSnippets = this.getEditableArea().find('.o_snippet_invisible');
+            const $invisibleSnippets = this.getEditableArea().find('.o_snippet_invisible').addBack('.o_snippet_invisible');
```

Comment:
Not needed anymore, I guess it was a bug induced by https://www.odoo.com/web#id=2122947&action=327&model=project.task&view_type=form&menu_id=4720 which is now reverted. But your fix will still be needed if the original problem of the mentioned task is solved another way. So maybe you should keep this but review your commit message :wink: